presto-cassandra/src/test/java/com/facebook/presto/cassandra/EmbeddedCassandra.java

```diff
@@ -50,6 +53,8 @@
     private static final String HOST = "127.0.0.1";
     private static final int PORT = 9142;
 
+    private static final Duration REFRESH_SIZE_ESTIMATES_TIMEOUT = new Duration(1, MINUTES);
```

Comment:
A minute is a while to wait for every tpch table to be refreshed for every test, does it actually take this long?